codebase/superdarn/src.idl/lib/main.1.25/rprm.pro

```diff
@@ -2,30 +2,24 @@
 ; ========
 ; Author: R.J.Barnes
 ; 
-; LICENSE AND DISCLAIMER
-; 
 ; Copyright (c) 2012 The Johns Hopkins University/Applied Physics Laboratory
 ; 
-; This file is part of the Radar Software Toolkit (RST).
-; 
 ; RST is free software: you can redistribute it and/or modify
-; it under the terms of the GNU Lesser General Public License as published by
+; it under the terms of the GNU General Public License as published by
 ; the Free Software Foundation, either version 3 of the License, or
-; any later version.
+; (at your option) any later version.
 ; 
-; RST is distributed in the hope that it will be useful,
+; This program is distributed in the hope that it will be useful,
 ; but WITHOUT ANY WARRANTY; without even the implied warranty of
-; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-; GNU Lesser General Public License for more details.
+; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+; GNU General Public License for more details.
 ; 
-; You should have received a copy of the GNU Lesser General Public License
-; along with RST.  If not, see <http://www.gnu.org/licenses/>.
+; You should have received a copy of the GNU General Public License
+; along with this program. If not, see <https://www.gnu.org/licenses/>.
 ; 
+; Modifications:
 ; 
-; 
-;
-; Public Functions
-; ----------------
+; Functions
 ;
```

Comment:
Similar to above, the "Public Functions" list header and underline format has been inadvertently lost from the original file.

Reply:
Nevermind, I think this answer the question.  It's not the same change to be done, but does touch all of the files.

Reply:
This should be resolved by 2382044.